requirements.txt

```diff
@@ -17,3 +17,4 @@ Fabric3==1.14.post1
 itsdangerous==0.24
 us>=1.0
 future
+email-validator==1.0.5
```

Comment:
ah hmm where do we use this dependency?

Reply:
Ahhh I see, this is brought in by latest WTForms now (I noticed via CI failing on #706). I'm with ya now